data/coalition/coalition missions.txt

```diff
@@ -160,7 +160,8 @@ mission "lost in coalition"
 	to offer
 		has "license: Coalition"
 	on offer
-		require "Jump Drive" 0
+		require
+			not "flagship attribute: jump drive"
```

Comment:
I believe these are older missions that predate this "flagship attribute" autocondition. The `require` node has been effectively replaced with the ability to use autoconditions that check your outfits in the `to *` node. What mechanically do we actually gain by turning the `require` node into effectively another condition set check?

Reply:
Yes. We have `to complete`, `to offer`, `to fail`, and `to accept`.